packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -1,100 +1,91 @@
-import dayjs from 'dayjs'
-import { useBlockNumber } from 'wagmi'
+import { useMedia } from 'react-use'
+import dayjs, { Dayjs } from 'dayjs'
 
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
-import { getBlockTime, getConfirmPeriodBlocks } from '../../util/networks'
-import {
-  NodeBlockDeadlineStatus,
-  NodeBlockDeadlineStatusTypes
-} from '../../hooks/arbTokenBridge.types'
-import { Tooltip } from './Tooltip'
+import { ChainId } from '../../util/networks'
 
-export function WithdrawalCountdown({
-  nodeBlockDeadline
-}: {
-  nodeBlockDeadline: NodeBlockDeadlineStatus
-}): JSX.Element {
-  const {
-    l1: { network: l1Network },
-    l2: { network: l2Network }
-  } = useNetworksAndSigners()
-  const { data: currentL1BlockNumber = 0 } = useBlockNumber({
-    chainId: l1Network.id,
-    watch: true
-  })
+/**
+ * Buffer for after a node is confirmable but isn't yet confirmed; we give 30 minutes, should usually/always be less in practice.
+ */
+const CONFIRMATION_BUFFER_MINUTES = 30
 
-  if (
-    typeof nodeBlockDeadline === 'undefined' ||
-    typeof l2Network === 'undefined'
-  ) {
-    return <span>Calculating...</span>
+function getTxRemainingMinutes({
+  createdAt,
+  parentChainId
+}: {
+  createdAt: string | Dayjs
```

Comment:
This component sends Dayjs object but in the next PR I will also need it as string to send data from local storage